src/components/Menu.vue

```diff
@@ -45,6 +45,7 @@ export default {
         null,
         'y'
       )
+      this.$emit('Menu-close')
```

Comment:
```suggestion
      this.$emit('menu-close')
```
コメントにもあったけど、統一化できたら良いね